src/java.base/share/classes/java/lang/Class.java

```diff
@@ -438,8 +438,15 @@ private static Class<?> forName(String className, Class<?> caller)
      * obtain any of the {@code Class} objects representing primitive
      * types or void.
      *
-     * <p> If {@code name} denotes an array class, the component type of
-     * the array class is loaded but not initialized.
+     * <p> If {@code name} denotes an array class whose element type
+     * is a class or interface, the name consists of one or more {@code '['}
+     * followed by the binary name of the element type and {@code ';'},
```

Comment:
There's an L between [ and component binary name, like [Ljava.lang.Object;

Reply:
Updated.